pyproject.toml.jinja

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = [
+  "setuptools>=67",
+  "setuptools_scm[toml]>=7.0",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{projectname}}"
+description = "{{description}}"
+authors= [{name="Scipp contributors"}]
+license = { "file" = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+{%- for version in range(min_python.split(".")[1] | int, max_python.split(".")[1] | int + 1) %}
+    "Programming Language :: Python :: {{version}}",
+{%- endfor %}
```

Comment:
👍 I thought there had to be a way..

Reply:
@jl-wynen This didn't work, it makes number 8, 9, 10, ..., without the major "3." bit.